logbook-core/src/test/java/org/zalando/logbook/CurlHttpLogFormatterTest.java

```diff
@@ -0,0 +1,61 @@
+package org.zalando.logbook;
+
+import org.junit.Test;
+import org.zalando.logbook.DefaultLogbook.SimpleCorrelation;
+import org.zalando.logbook.DefaultLogbook.SimplePrecorrelation;
+
+import java.io.IOException;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public final class CurlHttpLogFormatterTest {
+
+    @Test
+    public void shouldLogRequest() throws IOException {
+        final String correlationId = "c9408eaa-677d-11e5-9457-10ddb1ee7671";
+        final HttpRequest request = MockHttpRequest.create()
+                .withProtocolVersion("HTTP/1.0")
+                .withOrigin(Origin.REMOTE)
+                .withPath("/test")
+                .withQuery("limit=1")
+                .withHeaders(MockHeaders.of(
+                        "Accept", "application/json",
+                        "Content-Type", "text/plain"))
+                .withBodyAsString("Hello, world!");
```

Comment:
Could you add test with json payload which itself contains strings with single and double quotes?